### DISCUSSION TITLE
'Installation failed' when installing insiders via pip

### DISCUSSION THREAD

**Discussion by goddard82** [squidfunk/mkdocs-material#3029] [Q&A] [answered]
Hello, first of all apologies if this should be filed elsewhere but I think it's something I'm doing wrong rather than a bug.

When I follow the instructions to switch to Insiders using pip, I get "Installation Failed". I've created the access token beforehand.

I'm not sure what I'm doing wrong, so any help would be greatly appreciated.

Thanks 😊

**Comment by squidfunk** [comment]
Thanks for reporting. Did you follow the [installation guide](https://squidfunk.github.io/mkdocs-material/insiders/getting-started/)?  Also, do you have access to the private repository, i.e., did you or your organization sponsor the project? The `GH_TOKEN` must be linked to an account which was added as a collaborator to the repository.

**Comment by goddard82** [comment, marked as answer]
Hi Martin,

Yes, my organisation is a collaborator (Lockwood Publishing), and I followed the installation guide. My GitHub account is a collaborator so it should’ve worked.

I believe my manager, Mark Tully, has been in touch with you when setting up our sponsorship? Apologies if that’s wrong!

Thanks,

Dan

On 15 Sep 2021, at 14:43, Martin Donath ***@***.******@***.***>> wrote:



Thanks for reporting. Did you follow the installation guide<https://squidfunk.github.io/mkdocs-material/insiders/getting-started/>? Also, do you have access to the private repository, i.e., did you or your organization sponsor the project? The GH_TOKEN must be linked to an account which was added as a collaborator to the repository.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/squidfunk/mkdocs-material/discussions/3029#discussioncomment-1333500>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AKXCJD3WEZXXMNOF5G3CNU3UCCPIHANCNFSM5ECMSZ7Q>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Reply by squidfunk** [reply]
Ah yes, you're right. There are currently 200+ collaborators on this repository, my apologies for missing that fact. If you follow the guide that I linked in my previous comment and ensure that the associated token has the respective privileges, it will work. It's definitely a permission issue with the account accessing the repository.

**Comment by goddard82** [comment]
Resolved this now, I was attempting to install with pipenv rather than pip, which has worked. Thanks for your help 👍

On 15 Sep 2021, at 14:50, Martin Donath ***@***.******@***.***>> wrote:



Ah yes, you're right. There are currently 200+ collaborators on this repository, my apologies for missing that fact. If you follow the guide that I linked in my previous comment and ensure that the associated token has the respective privileges, it will work. It's definitely a permission issue with the account accessing the repository.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/squidfunk/mkdocs-material/discussions/3029#discussioncomment-1333556>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AKXCJDYTUFTIIGYHBEX5WN3UCCQA7ANCNFSM5ECMSZ7Q>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
